scripts/website/build-examples.sh

```diff
@@ -12,6 +12,6 @@ for example in examples/*; do
     name=$(basename "$example")
     echo "Building $name example..."
 
-    (cd "$example" && yarn && yarn build && rsync -r "dist/" "../../website/examples/$name/")
```

Comment:
what's this change for?

Reply:
`rsync` is unavailabled on CircleCI and we need to execute this script on it before running the tests.